projects/systelab-translate/src/lib/date-util/date-util.ts

```diff
@@ -251,4 +163,64 @@ export class DateUtil {
 		const locale: Locale = localeForSystelabLocale.get(localeString)
 		return locale ? locale : enUS;
 	}
+
+	private getLocalizedStringDateFormat(locale: string, isFullYear = false): string {
+		const year = isFullYear ? 'yyyy' : 'yy';
+		switch (locale) {
+			case 'en-US':
+				return isFullYear ? `MM/dd/${year}` : `M/d/${year}`;
```

Comment:
It is confussing to use the isFullYear variable to control if the month and day comes with two or maybe one digit.